ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productteaser/v1/productteaser/productteaser.html

```diff
@@ -16,16 +16,27 @@
 
 <sly data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html"
      data-sly-use.product="com.adobe.cq.commerce.core.components.models.productteaser.ProductTeaser">
+    <sly data-sly-call="${clientlib.all @ categories='core.cif.components.productteaser.v1'}"></sly>
     <sly data-sly-test.ispropavailable="${properties.selection}">
-        <div data-sly-test.isvalid="${product.url}" class="item__root">
+        <div data-sly-test.isvalid="${product.url}" class="item__root" data-cmp-is="productteaser">
             <a class="item__images" href=${product.url}>
                 <img class="item__image" width="100%" height="100%"
                      src="${product.image}" alt="${product.image}"/>
-            </a>
+            </a>                                
             <a class="item__name" href=${product.url}><span>${product.name}</span></a>
             <div class="item__price">
                 <span> ${product.formattedPrice} </span>
             </div>
+            <div class="productteaser__cta" data-sly-test="${product.callToAction != null}">
+                <button data-sly-test="${product.callToAction == 'add-to-cart'}" data-action="addToCart" data-item-sku="${product.sku}"
```

Comment:
These buttons would be an optimal candidate for moving into a separate HTL file in case a customer wants to customise cand extend the call to action choices, right?

Reply:
Will do